services/motion/builtin/replanner.go

```diff
@@ -11,40 +14,44 @@ type replanResponse struct {
 	replan bool
 }
 
+// replanFn is an alias for a function that will be polled by a replanner.
+type replanFn func(context.Context, [][]referenceframe.Input, int) (bool, error)
+
 // replanner bundles everything needed to execute a function at a given interval and return.
 type replanner struct {
 	period       time.Duration
-	fnToPoll     func(ctx context.Context) replanResponse
+	needReplan   replanFn
```

Comment:
Can we add a comment on this? I find this less intuitive than the original name.

Reply:
I kind of like the original name too now that I think of it.  I will change back

Reply:
Actually the reason I changed it is because I changed the type to return a bool and it is more descriptive what that bool means if I name it this way.  Adding a comment